pom.xml

```diff
@@ -31,6 +31,13 @@
             <artifactId>selenium-java</artifactId>
             <version>4.7.2</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/org.testng/testng -->
```

Comment:
You don't need to add again the testng dependency as this is already in pom file at lines 18-23. Please remove these lines added by you now. Thanks!

Reply:
done